libsolidity/formal/CHC.cpp

```diff
@@ -1584,6 +1638,32 @@ optional<string> CHC::generateCounterexample(CHCSolverInterface::CexGraph const&
 				auto const& outParams = calledFun->returnParameters();
 				if (auto outStr = formatVariableModel(outParams, outValues, "\n"); !outStr.empty())
 					localState += outStr + "\n";
+
+				optional<unsigned> localErrorId;
+				solidity::util::BreadthFirstSearch<unsigned> bfs{{summaryId}};
+				bfs.run([&](auto _nodeId, auto&& _addChild) {
+					auto const& edges = _graph.edges.at(_nodeId);
+					if (edges.size() == 1)
+					{
+						auto const* child = nodePred(edges.front());
+						if (child->isFunctionErrorBlock())
+						{
+							localErrorId = edges.front();
+							bfs.abort();
+						}
+					}
+					for (auto c: edges)
+						_addChild(c);
+				});
+
+				if (localErrorId.has_value())
+				{
+					auto const* localError = nodePred(*localErrorId);
+					solAssert(localError && localError->isFunctionErrorBlock(), "");
+					auto [localValues, localVars] = localError->localVariableValues(nodeArgs(*localErrorId));
```

Comment:
```suggestion
					auto const& [localValues, localVars] = localError->localVariableValues(nodeArgs(*localErrorId));
```

Reply:
The values are moved out of the function, so not much use it making it a `ref`, but I made it `const`.